resources/qml/WelcomePages/WhatsNewContent.qml

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2019 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+
+import UM 1.3 as UM
+import Cura 1.1 as Cura
+
+
+//
+// This component contains the content for the "What's new in Ultimaker Cura" page of the welcome on-boarding process.
+//
+Item
+{
+    UM.I18nCatalog { id: catalog; name: "cura" }
+
+    Label
+    {
+        id: titleLabel
+        anchors.top: parent.top
+        anchors.topMargin: 40
+        anchors.horizontalCenter: parent.horizontalCenter
+        horizontalAlignment: Text.AlignHCenter
+        text: catalog.i18nc("@label", "What's new in Ultimaker Cura")
+        color: UM.Theme.getColor("primary_button")
+        font: UM.Theme.getFont("large_bold")
+        renderType: Text.NativeRendering
+    }
+
+    Rectangle
+    {
+        anchors.top: titleLabel.bottom
+        anchors.bottom: getStartedButton.top
+        anchors.topMargin: 40
+        anchors.bottomMargin: 40
+        anchors.horizontalCenter: parent.horizontalCenter
+        width: parent.width * 3 / 4
```

Comment:
Why 3/4ths? Also, this needs to be rounded to double pixels because we're centring it and it contains text.

Reply:
Rounded in 6f911f992. Same reason as the ones in the other pages. This looks better in the sense that the wall text seems to have a better left and right margin.